README.md

```diff
@@ -30,3 +30,21 @@ See [the wiki](https://github.com/GoogleChrome/chrome-types/wiki) for more.
 Running the code requires Node 16+ as well as a working version of Python (3 is preferred, but 2.7+ should work) installed on your system.
 This has only been tested on Linux and macOS.
 Python is used to convert Chromium's internal IDL format to JSON.
+
+**Steps**
+
+1. Add library as a dependency.
+   ```bash
+   npm i chrome-types --save-dev
+   ```
+3. Update compiler options in `tsconfig.json`
+
+    ```patch
+    {
+        "compilerOptions": {
+          ...
+    +     "types": ["chrome-types"]
+          ...
+        },
+    }
+    ```
```

Comment:
```suggestion

**Steps**

1. Add library as a dependency

    ```bash
    npm install --save-dev chrome-types
    ```

2. Add `chrome-types` to the [type roots](https://www.typescriptlang.org/tsconfig#typeRoots) in `tsconfig.json`

    ```diff
    {
        "compilerOptions": {
    +     "typeRoots": [
    +       "./node_modules/@types",
    +       "./node_modules/chrome-types"
    +     ]
        }
    }
    ```
```

It's probably a good idea not to exclude other installed types: [`types`](https://www.typescriptlang.org/tsconfig#types), [`typeRoots`](https://www.typescriptlang.org/tsconfig#typeRoots)

Reply:
I think it would be nice to include either the default node_modules directly, or a note that you need to do that. Would you be ok adding that @tusharmath?

Reply:
> I think it would be nice to include either the default node_modules directly, or a note that you need to do that. Would you be ok adding that @tusharmath?

I agree that a note is appropriate.